test_runner/regress/test_broken_timeline.py

```diff
@@ -75,7 +77,7 @@ def test_local_corruption(neon_env_builder: NeonEnvBuilder):
     # (We don't check layer file contents on startup, when loading the timeline)
     #
     # This will change when we implement checksums for layers
-    with pytest.raises(Exception, match="get_values_reconstruct_data for layer ") as err:
+    with pytest.raises(Exception, match="failed to get checkpoint bytes") as err:
```

Comment:
Can the "get_values_reconstruct_data for layer" message be removed from allowed_errors now?

Reply:
No we can't:
```
2025-02-10T14:04:13.884264Z  WARN initial_size_calculation{tenant_id=14ad3bbd452c356bac506cd7e8c92589 shard_id=0000 timeline_id=29b6ac6946b7f25139f6091ab5495113}: initial size calculation failed: PageRead(Other(get_values_reconstruct_data for layer 000000000000000000000000000000000000-FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF__00000000014AB4B1-00000000014E9CB9-00000001
```